cmd/linkflags/main.go

```diff
@@ -54,7 +57,11 @@ func run() error {
 	log.SetFlags(0)
 	flag.Parse()
 	if *pkg == "" {
-		return fmt.Errorf("-pkg required")
+		dir, err := filepath.Abs(filepath.Dir(os.Args[0]))
```

Comment:
I was wondering if any intermediate symlinks will interfere? Will it still work from a symlinked location?